### PR TITLE
Fixes #9811 - Add keyhandler article

### LINKED ISSUE #9811: Using dynamic help: mention how to enable Alt-based keyboard shortcuts on macOS
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/learn/shell/dynamic-help.md

### Summary

While it is true that <kbd>Alt</kbd>-based keyboard shortcuts do not work on macOS _by default_, it is easy to configure the terminal applications (the built-in Terminal.app and popular third-party alternative iTerm2.app) to support them.

This is worth mentioning as an alternative to resorting to <kbd>Ctrl</kbd>-based redefinitions.

See https://github.com/PowerShell/PSReadLine/issues/1075#issuecomment-1412449026

### Details

_No response_

### Suggested Fix

_No response_

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Add keyhandler article

- Fixes [AB#79955](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/79955)
- Fixes #9811

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/docs-conceptual/learn/shell/using-keyhandlers.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+description: This article explains how to use PSReadLine keybindings for various platforms and terminals applications.
+title: Using PSReadLine key handlers
+ms.date: 03/27/2023
+---
+# Using PSReadLine key handlers
+
+In **PSReadLine**, key handlers are configuration settings that map **PSReadLine** functions to
```
Comment (StevenBucher98) [inline reference/docs-conceptual/learn/shell/using-keyhandlers.md:8]: I am not sure I like saying `In PSReadLine` as it sorta implies it a particular environment or mode of PowerShell.

Comment (StevenBucher98) [inline reference/docs-conceptual/learn/shell/using-keyhandlers.md:8]:
```suggestion
The **PSReadLine** module in PowerShell adds key handlers which are configuration settings that map **PSReadLine** functions to
```

File: reference/docs-conceptual/learn/shell/using-keyhandlers.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+**PSReadLine** has several predefined key handlers that are bound by default. You can also define
+your own custom key handlers. Run the following command to list the key handlers that are currently
+defined.
+
+```powershell
+Get-PSReadLineKeyHandler
+```
+
+You can also get a list of all unbound functions that are available to be bound by a key handler.
```
Comment (StevenBucher98) [inline reference/docs-conceptual/learn/shell/using-keyhandlers.md:24]:
```suggestion
You can also get a list of all unbound **PSReadLine** functions that are available to be bound by a key handler.
```

File: reference/docs-conceptual/learn/shell/using-keyhandlers.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/learn/shell/using-keyhandlers.md:8]: PSReadLine is the only way you get the key handlers.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**